app/uk/gov/hmrc/cgtpropertydisposals/service/returns/ReturnsService.scala

```diff
@@ -113,6 +123,79 @@ class DefaultReturnsService @Inject() (
     } yield returnResponse
   }
 
+  private def deltaCharge(
+    response: DesSubmitReturnResponse,
+    cgtReference: CgtReference,
+    fromDate: LocalDate,
+    toDate: LocalDate
+  )(implicit hc: HeaderCarrier): EitherT[Future, Error, Option[DeltaCharge]] =
+    response.ppdReturnResponseDetails.chargeReference match {
```

Comment:
if the return is not an amend return we can return `None` straight away - delta charges only apply to amends